algoliasearch/types_cluster.go

```diff
@@ -0,0 +1,8 @@
+package algoliasearch
+
+type Cluster struct {
+	ClusterName string `json:"clusterName"`
+	NbRecords   int    `json:"nbRecords"`
+	NbUserIDs   int    `json:"nbUserIDs"`
+	DataSize    int    `json:"dataSize"`
+}
```

Comment:
Why do you think of putting all the types related to MCM under `types_mcm.go`? I find it clearer IMHO.

Reply:
Definitely 👍